run_scripts/test_contacts.json

```diff
@@ -0,0 +1,4 @@
+[
```

Comment:
Can we rename this test_contact_rapid_pro_ids.json? Otherwise it's bit hard to know what these GUIDs are without reading the code.